cluster-api/cmd/machineset-controller/README.md

```diff
@@ -0,0 +1,28 @@
+# MachineSet Controller
+is an initial implementation of `MachineSet` controller. It watches for `MachineSet` resources
```

Comment:
I'd remove "initial" as this will evolve (and then we don't need to update the README.md :-).

Reply:
okay.